.github/workflows/release.yaml

```diff
@@ -1,21 +1,36 @@
 ---
-name: Release Container Image
+name: Release
 
 on:
   push:
     branches: [main]
-    tags:
-      - "v*.*.*"
+  workflow_call:
```

Comment:
i already runs on main after we merge this. when would we need this manual trigger?

Reply:
i'm not sure, copy pasted it from https://github.com/adfinis/outdated

Reply:
i removed it